README.md

```diff
@@ -1,11 +1,9 @@
 # `zq` [![CI][ci-img]][ci] [![GoDoc][doc-img]][doc]
 
-`zq` is a command-line tool that's useful for searching and analyzing logs;
+`zq` is a command-line tool for searching and analyzing logs,
```

Comment:
This is pretty close to the repo tag line "Command-line processor for structured logs", but we should ideally align them completely. (tool vs processor? logs vs structured logs?)